central/complianceoperator/v2/scanconfigurations/service/service_impl.go

```diff
@@ -47,7 +47,7 @@ var (
 		},
 	})
 
-	configNameRegexp = regexp.MustCompile(`^[a-z0-9][a-z0-9.-]*[a-z0-9]?$`)
+	configNameRegexp = regexp.MustCompile(`^[a-z0-9][a-z0-9.-]*$`)
```

Comment:
@lvalerom 
I've shortened this Regex because the shorter version is logically equivalent to the longer one, but this makes me wonder if the Regex maybe was incorrect in the first place. Do you know the rules for config names we are trying to enforce here?